jax/numpy/lax_numpy.py

```diff
@@ -11,7 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
Can you revert this file for now? I like the idea of auto-formatting, but let's do one thing at a time :)